deployments/common/ap-policy-definition.yaml

```diff
@@ -0,0 +1,874 @@
+                        - Spring Boot
+                        - CouchDB
+                        - SQLite
+                        - Handlebars
+                        - Mustache
+                        - Prototype
+                        - Zend
+                        - Redis
+                        - Underscore.js
+                        - Ember.js
+                        - ZURB Foundation
+                        - ef.js
+                        - Vue.js
+                        - UIKit
+                        - TYPO3 CMS
+                        - RequireJS
+                        - React
+                        - MooTools
+                        - Laravel
+                        - GraphQL
+                        - Google Web Toolkit
+                        - Express.js
+                        - CodeIgniter
+                        - Backbone.js
+                        - AngularJS
+                        - JavaScript
+                        - Nginx
+                        - Jetty
+                        - Joomla
+                        - JavaServer Faces (JSF)
+                        - Ruby
+                        - MongoDB
+                        - Django
+                        - Node.js
+                        - Citrix
+                        - JBoss
+                        - Elasticsearch
+                        - Apache Struts
+                        - XML
+                        - PostgreSQL
+                        - IBM DB2
+                        - Sybase/ASE
+                        - CGI
+                        - Proxy Servers
+                        - SSI (Server Side Includes)
+                        - Cisco
+                        - Novell
+                        - Macromedia JRun
+                        - BEA Systems WebLogic Server
+                        - Lotus Domino
+                        - MySQL
+                        - Oracle
+                        - Microsoft SQL Server
+                        - PHP
+                        - Outlook Web Access
+                        - Apache/NCSA HTTP Server
+                        - Apache Tomcat
+                        - WordPress
+                        - Macromedia ColdFusion
+                        - Unix/Linux
+                        - Microsoft Windows
+                        - ASP.NET
+                        - Front Page Server Extensions (FPSE)
+                        - IIS
+                        - WebDAV
+                        - ASP
+                        - Java Servlets/JSP
+                        - jQuery
+                        type: string
+                    type: object
+                  type: array
+                signature-sets:
+                  items:
+                    properties:
+                      alarm:
+                        type: boolean
+                      block:
+                        type: boolean
+                      name:
+                        enum:
```

Comment:
Custom signature sets are allowed to be defined, not just the enum list of factory sets.